tools/renovate/README.md

```diff
@@ -7,6 +7,23 @@
 
 Otter framework provides a set of Renovate presets to facilitate the setup and reduce the boilerplate in your `.renovaterc.json`
 
+## Available presets
+
+| Preset                        | Parameters                         | Description                                                                                        |
+|-------------------------------|------------------------------------|----------------------------------------------------------------------------------------------------|
+| **group/design-factory**      |                                    | Group the dependencies related to Design Factory .                                                 |
```

Comment:
```suggestion
| **group/design-factory**      |                                    | Group the dependencies related to Design Factory.                                                 |
```

Reply:
Also, to be consistent, either add or remove the "." for each line (not the case currently)